prow/github/types.go

```diff
@@ -943,13 +943,10 @@ type DraftReviewComment struct {
 	Body     string `json:"body"`
 }
 
-// Content contains information about a github file/directory.
+// Content is some base64 encoded github file content
```

Comment:
Can we link to the piece of doc that shows what GH response does this represent?